From 74979c4e02aec3c9d9a4f2a7cdbba3fb0ae0b881 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sat, 20 May 2023 15:52:32 +0200
Subject: [PATCH 391/406] Revert "arm64: dts: rk3399-pinephone-pro: Add correct
 camera rotation values"

This reverts commit 0ee732cb60bc498ee51d8697dc236e07e54b0d4f.
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index 5dcff7a72d2b..e52e26973d3f 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -890,7 +890,7 @@ wcam: camera@1a {
 		reset-gpios = <&gpio1 RK_PA0 GPIO_ACTIVE_LOW>;
 		powerdown-gpios = <&gpio2 RK_PD4 GPIO_ACTIVE_HIGH>;
 
-		rotation = <270>;
+		rotation = <180>;
 
 		lens-focus = <&wcam_lens>;
 		flash-leds = <&sgm3140_flash>;
@@ -930,7 +930,7 @@ ucam: camera@36 {
 		rockchip,camera-module-name = "CameraKing";
 		rockchip,camera-module-lens-name = "Largan-9569A2";
 
-		rotation = <90>;
+		rotation = <180>;
 
 		port {
 			ucam_out: endpoint {
-- 
2.34.1

